action-templates/actions/action-checkout/action.yml

```
@@ -1,8 +1,27 @@
+  javascript-typescript-vue:
+    description: "name of the artifact upload"
+    value: ${{ steps.filter.outputs.javascript-typescript-vue }}
+  python:
+    description: "name of the artifact upload"
+    value: ${{ steps.filter.outputs.python }}
+    
 runs:
   using: "composite"
   steps:
     - name: Checkout code
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)